PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterCharacterScripts/Sound.rbxmx

```diff
@@ -0,0 +1,381 @@
+<roblox xmlns:xmime="http://www.w3.org/2005/05/xmlmime" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.roblox.com/roblox.xsd" version="4">
+	<External>null</External>
+	<External>nil</External>
+	<Item class="Script" referent="RBXc83eeca86f73463493b2478b693fc06d">
+		<Properties>
+			<bool name="Disabled">false</bool>
+			<Content name="LinkedSource"><null></null></Content>
+			<string name="Name">Sound</string>
+			<string name="ScriptGuid"></string>
+			<ProtectedString name="Source"><![CDATA[--[[
+	This script creates sounds which are placed under the character head.
+	These sounds are used by the "LocalSound" script.
+]]--
+--humanoidSoundNewLocal.rbxmx--
+
+function CreateNewSound(name, id, looped, pitch)
+	local sound = Instance.new("Sound")
+	sound.SoundId = id
+	sound.Name = name
+	sound.archivable = false
+	sound.Parent = script.Parent.Head
+	sound.Pitch = pitch
+	sound.Looped = looped	
+	
+	return sound
+end
+
+CreateNewSound("GettingUp", 	"rbxasset://sounds/action_get_up.mp3",					false, 	1)
+CreateNewSound("Died", 		"rbxasset://sounds/uuhhh.mp3",								false, 	1) 
+CreateNewSound("FreeFalling", 	"rbxasset://sounds/action_falling.mp3",					true,	1)
+CreateNewSound("Jumping", 		"rbxasset://sounds/action_jump.mp3",					false,	1)
+CreateNewSound("Landing", 		"rbxasset://sounds/action_jump_land.mp3",				false,	1)
+CreateNewSound("Splash", 		"rbxasset://sounds/impact_water.mp3",					false, 	1)
+CreateNewSound("Running", 		"rbxasset://sounds/action_footsteps_plastic.mp3",		true, 	1.85)
+CreateNewSound("Swimming", 	"rbxasset://sounds/action_swim.mp3",						true,	1.6)
+CreateNewSound("Climbing", 	"rbxasset://sounds/action_footsteps_plastic.mp3",			true,	1)]]></ProtectedString>
+		</Properties>
+		<Item class="LocalScript" referent="RBX35b0a371f02e4ae2baca5961c8b67265">
+			<Properties>
+				<bool name="Disabled">false</bool>
+				<Content name="LinkedSource"><null></null></Content>
+				<string name="Name">LocalSound</string>
+				<string name="ScriptGuid"></string>
+				<ProtectedString name="Source"><![CDATA[--[[
+	Author: @spotco
+	This script runs locally for the player of the given humanoid.
+	This script triggers humanoid sound play/pause actions locally.
+	
+	The Playing/TimePosition properties of Sound objects bypass FilteringEnabled, so this triggers the sound
+	immediately for the player and is replicated to all other players.
+	
+	This script has been optimized to reduce network traffic (in comparison to the existing humanoid sound scripts.
+]]--
+
+--All sounds are referenced by this ID
+local SFX = {
+	Died = 0;
+	Running = 1;
+	Swimming = 2;
+	Climbing = 3,
+	Jumping = 4;
+	GettingUp = 5;
+	FreeFalling = 6;
+	FallingDown = 7;
+	Landing = 8;
+	Splash = 9;
+}
+
+local Humanoid = nil
+local Head = nil
+
+--SFX ID to Sound object
```

Comment:
Thinking about it:
This is to keep the SFX.FreeFalling syntax (as opposed to SFX["FreeFalling"]).
The first would be better in a typed language because of compiler checking, I'm guessing the errors in lua would be about the same.
Anyway, I'm now for keeping it as is (SFX.FallingDown) because easier to read?